notes/api.md

```diff
@@ -128,6 +128,10 @@ URL query parameters may be specified to narrow the search:
 - `end=YYYY-MM-DD`: end date
```

Comment:
For date formats do we want to provide more than one just in case? YYYYMMDD as well? ( know XX/ZZ/YYYY raises issues)

Reply:
Nooooo!  😢 Let's use the ISO format only.